cmd/neofs-cli/internal/common/netmap.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import (
+	"encoding/hex"
+
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+	"github.com/spf13/cobra"
+)
+
+func PrettyPrintNodeInfo(cmd *cobra.Command, node *netmap.NodeInfo,
```

Comment:
Can we accept `node` parameter by value? Seems like we don't modify the instance.

Also I suggest to write brief docs about function: it's not really clear what `index`, `indent`, and `short` fields do looking only at func's signature.

Reply:
Thanks, docs added.